libminifi/include/utils/TimeUtil.h

```diff
@@ -37,6 +37,24 @@ namespace minifi {
 namespace utils {
 namespace timeutils {
 
+/**
+ * Converts the time point to the elapsed time since epoch
+ * @returns TimeUnit since epoch
+ */
+template<typename TimeUnit, typename TimePoint>
+uint64_t getTimestamp(const TimePoint& time_point) {
+  return std::chrono::duration_cast<TimeUnit>(time_point.time_since_epoch()).count();
+}
+
+/**
+ * Converts the time since epoch into a time point
+ * @returns the time point matching the input timestamp
+ */
+template<typename TimeUnit, typename ClockType>
+std::chrono::time_point<ClockType> getTimePoint(uint64_t timestamp) {
+  return std::chrono::time_point<ClockType>() + TimeUnit(timestamp);
+}
```

Comment:
I criticized these in your other PR, which would apply here, too. If you think that they are useful, feel free to reintroduce them there, but if they don't improve the readability, you may want to consider removing them.

Reply:
sure thing :+1: , I removed these in https://github.com/apache/nifi-minifi-cpp/pull/1219/commits/a0e996b4844c92dd3d2cc4d70955ad4ea5697f31#diff-4a76905d55704437ae0a7a4ee434a73f057c105c97bdf8756f5747b4fbc65e9fL40-L57